frontend/cypress/support/helper/dom-helper/MatSelectHelper.ts

```diff
@@ -0,0 +1,20 @@
+import { PageObjectMapperBase } from './pageObjectMapperBase';
+
+export default class MatSelectHelper extends PageObjectMapperBase {
+  validatePage(): void {
+    // Does not need to be implemented this comment is for making linter happy
```

Comment:
```suggestion
    // Does not need to be implemented this comment is for making the linter happy
```
Pendantic but why not.